src/components/Card.tsx

```diff
@@ -0,0 +1,42 @@
+import Image from "next/image";
+import { ChevronRightIcon } from "@heroicons/react/24/outline";
+import { PhotoIcon } from "@heroicons/react/24/outline";
+import type { Article } from "@/types/article";
+
+type Props = {
+  article: Article;
+};
+
+const Card = ({ article }: Props) => {
+  return (
+    <a
+      href={article.web_url}
+      target="_blank"
+      rel="noreferrer noopener"
+      key={article._id}
+      className="mb-4 pb-4 border-b border-gray-300 grid grid-cols-[auto_1fr_auto] md:items-center gap-8"
+    >
+      <figure className="w-28 h-[75px] bg-gray-300 overflow-hidden flex items-center justify-center relative">
+        {article.multimedia?.default?.url ? (
+          <>
+            <Image
+              src={article.multimedia?.default?.url || ""}
+              alt={article.headline.main}
+              fill={true}
+              sizes="112px"
+            />
+          </>
+        ) : (
+          <PhotoIcon className="w-8 h-8 text-gray-400" />
+        )}
+      </figure>
+      <div>
+        <h3 className="text-lg font-bold">{article.headline.main}</h3>
+        <p className="text-sm">{article.abstract}</p>
+      </div>
+      <ChevronRightIcon className="w-5 h-5 text-gray-400" />
+    </a>
```

Comment:
responsiveをつけているのは重要なのでとても良いです！

一例ですが、レスポンシブにしたい場合、こんな感じで描くことができます 😉 
- gridで大きさが固定されてしまっているので1frで可変するように変更
- 親の要素にw-full, h-auto aspect ratioを追加

```suggestion
   <a
      href={article.web_url}
      target="_blank"
      rel="noreferrer noopener"
      key={article._id}
      className="mb-4 pb-4 border-b border-gray-300 grid grid-cols-[1fr_1fr_auto] md:items-center gap-8"
    >
      <figure className="w-full h-auto aspect-[112/75] bg-gray-300 overflow-hidden flex items-center justify-center relative">
        {article.multimedia?.default?.url ? (
          <>
            <Image
              src={article.multimedia?.default?.url || ""}
              alt={article.headline.main}
              fill={true}
              sizes="112px"
              className="object-cover"
            />
          </>
        ) : (
          <PhotoIcon className="w-8 h-8 text-gray-400" />
        )}
      </figure>
      <div>
        <h3 className="text-lg font-bold">{article.headline.main}</h3>
        <p className="text-sm">{article.abstract}</p>
      </div>
      <ChevronRightIcon className="w-5 h-5 text-gray-400" />
    </a>
```

Reply:
ありがとうございます！grid-cols-[1fr_4fr_auto]にして画像も可変するようにしました！